### ISSUE TITLE
Gerber files

### ISSUE THREAD

**Issue by jeroentaverne** [retro16/acsi2stm#19]
Hi. Are there some gerber files available for this project? I know it can be hand wired, but a real PCB will make it look much nicer. Thanks!

**Comment by tomtebloss** [comment]
I did fork this out and made some PCB: https://github.com/tomtebloss/acsi2stm/tree/master/gerbers

**Comment by luciust** [comment]
Hey, thanks for the gerbers, just a note: the R3 is 10k not 100k. With 100k you can't set the programming mode. BOM needs fixing pictures are showing correct resistor.

**Comment by tomtebloss** [comment]
Thx for pointing it out. Changed BOM and readme. I changed R4 to 10K as well since it is also part of the programming mode but not really used in out project.

**Comment by luciust** [comment]
@tomtebloss No problemo, big thanks for gerbers - I've finished soldering first one and now trying to cope with my STM32 version which is odd - seems to be either not starting the official build or my compiled version, but kind-of works (shows up as multiple ADAPTEC 5500 controlers) with this binary:  https://github.com/retro16/acsi2stm/issues/15#issuecomment-968469499

**Comment by luciust** [comment]
@tomtebloss One more question - the R2 on your photo is 1K while BOM states 10k. Should it be 1k as well?

**Comment by tomtebloss** [comment]
> @tomtebloss One more question - the R2 on your photo is 1K while BOM states 10k. Should it be 1k as well?

I'm unsure which photo you mean? Schematics says 10K and the [photo](https://github.com/tomtebloss/acsi2stm/blob/master/gerbers/images/ACSI2STM_v1.2_back.jpg) is 1002? A 10K right?

Anyway, it's part of the reset circuit along with C2.

**Comment by luciust** [comment]
@tomtebloss Yeah - correct the 4 digit mode - yet if photo is correct - then BOM says: R2, R3 are 10k , while on the said picture we have R2 @ 10k and R4 @ 100k. My fault I got confused with my 3 and your 4 digit code - but I would say that BOM and R2,R3 and R4 vs picture - need your eyes. I'm debugging the fake STM32 - thus looking at each small piece on blue pill vs gerber too...

**Comment by tomtebloss** [comment]
Here is the [schematics](https://stm32-base.org/assets/pdf/boards/original-schematic-STM32F103C8T6-Blue_Pill.pdf) I used (if I remember correctly) of the "bluepill" when I designed my board. Apparently R10 should be changed to 1.5K. So I will probably change that at least. I didn't use USB for programming. Only for power.

I will have to change some resistors and try mine out first and then take new photos.
